scripts/test/Muon/corrections_tab_widget/background_corrections_model_test.py

```diff
@@ -0,0 +1,182 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+from unittest import mock
+
+from mantid.api import AnalysisDataService, FrameworkManager
+from mantid.simpleapi import CreateSampleWorkspace
+
+from Muon.GUI.Common.corrections_tab_widget.corrections_model import CorrectionsModel
+from Muon.GUI.Common.corrections_tab_widget.background_corrections_model import (BackgroundCorrectionsModel,
+                                                                                 DEFAULT_START_X, DEFAULT_END_X,
+                                                                                 DEFAULT_X_LOWER, DEFAULT_X_UPPER)
+from Muon.GUI.Common.test_helpers.context_setup import setup_context
+
+
+class BackgroundCorrectionsModelTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        FrameworkManager.Instance()
+
+    def setUp(self):
+        context = setup_context()
+        self.corrections_model = CorrectionsModel(context.data_context, context.corrections_context)
+        self.model = BackgroundCorrectionsModel(self.corrections_model, context, context.corrections_context)
+
+        self.runs = ["84447", "84447", "84447", "84447"]
+        self.groups = ["fwd", "bwd", "top", "bottom"]
```

Comment:
for PSI data you are more likely to have "F, B, L, R" as the group names (they often only use a single letter). Dont worry about changing anything. Thought it would be good for you to know.